tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -781,38 +781,92 @@ def _search(self, experiment_id, filter_string=None,
         return [r.info.run_id
                 for r in self.store.search_runs(exps, filter_string, run_view_type, max_results)]
 
-    def test_order_by_metric(self):
+    def get_ordered_runs(self, order_clauses, experiment_id):
+        return [
+            r.data.tags[mlflow_tags.MLFLOW_RUN_NAME]
+            for r in self.store.search_runs(experiment_ids=[experiment_id],
+                                            filter_string="",
+                                            run_view_type=ViewType.ALL,
+                                            order_by=order_clauses)]
+
+    def test_order_by_metric_tag_param(self):
```

Comment:
I tried running this test against a MySQL database, and it emitted the following warning several times:
```
tests/store/tracking/test_sqlalchemy_store.py::TestSqlAlchemyStoreMysqlDb::test_order_by_metric_tag_param
  /Users/czumar/anaconda2/lib/python2.7/site-packages/sqlalchemy/sql/base.py:556: SAWarning: Column 'CASE WHEN (anon_1.is_nan IS true) THEN :param_1 WHEN (anon_1.value IS NULL) THEN :param_2 ELSE :param_3 END' on table <sqlalchemy.sql.selectable.Select at 0x1190bebd0; Select object> being replaced by <sqlalchemy.sql.elements.ColumnClause at 0x118e17e10; %(4716925584 anon)s>, which has the same key.  Consider use_labels for select() statements.
    % (key, getattr(existing, "table", None), value)
```

Can you implement the `use_labels` suggestion to remove column name conflicts between tables? This is an argument to `select()` (https://docs.sqlalchemy.org/en/13/core/selectable.html#sqlalchemy.sql.expression.select).